pkg/controller/issueapi/parse_date.go

```diff
@@ -50,6 +56,41 @@ func init() {
 	}
 }
 
+func (c *Controller) parseDate(d string, tzOffset int, parseSettings *dateParseSettings) (*time.Time, *issueResult) {
```

Comment:
I don't think this should be coupled with the issueResult. Just return `*time.Time, error` and handle the rest upstream.